editor/src/components/navigator/navigator.tsx

```diff
@@ -140,6 +133,41 @@ export const NavigatorComponent = betterReactMemo('NavigatorComponent', () => {
     dispatch([EditorActions.togglePanel('navigator')])
   }, [dispatch])
 
+  const Item = React.memo(({ index, style }: ListChildComponentProps) => {
```

Comment:
please always use `betterReactMemo` instead of `React.memo`

Reply:
Good shout, fixed in edb21800.